boards/arm/cxd56xx/spresense/scripts/Make.defs

```diff
@@ -35,6 +35,7 @@
 
 include ${TOPDIR}/.config
 include ${TOPDIR}/tools/Config.mk
+include ${TOPDIR}/boards/arm/cxd56xx/scripts/cxd56xx_Config.mk
```

Comment:
How about remove cxd56xx_ prefix? because:
1.The path already have cxd56xx
2.Align with tools/Config.mk

Reply:
Okay.  Doing that now.  Should also change zds_Config.mk for the same reason.  I was following the pattern set by zds_Config.mk.  I will push new changes momentarily.